src/Extensions/DataListExtension.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace SilverStripe\Versioned\Extensions;
+
+use RuntimeException;
+use SilverStripe\Core\Extension;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\DataList;
+use SilverStripe\Versioned\Versioned;
+
+/**
+ * @extends Extension<DataList>
+ */
+class DataListExtension extends Extension
+{
+    public function onPrepopulateCaches(array $ids): void
```

Comment:
```suggestion
    protected function onPrepopulateCaches(array $ids): void
```
Sorry, I should have noticed this the first time around.

Reply:
Updated